openchs-server-api/src/main/java/org/openchs/service/IndividualService.java

```diff
@@ -32,6 +32,17 @@ public IndividualService(ConceptRepository conceptRepository, IndividualReposito
         this.individualRepository = individualRepository;
     }
 
+    public  IndividualContract getSubjectEncounters(String individualUuid){
+        Individual individual = individualRepository.findByUuid(individualUuid);
+        if (!Objects.nonNull(individual)) {
```

Comment:
Is there any reason we are using Objects.nonNull or Objects.isNull? Will not just `individual == null` or `individual != null` suffice?